blog/rome-incentives/README.md

```diff
@@ -0,0 +1,225 @@
+
+Basically, the payment per item is constant until the number of items in
+category multiplied by maximum payment per item exceeds the maximum weekly
+pool. We have chosen this reward structure in order to incentivize users to
+upload a variety of different content types on Rome.
+
+Payouts for this role will be made every Monday at ~11:00GMT.
+
+Content Curators
+----------------
+
+![](Content%20Curators.png)
+
+<i>Earn your share of $375 per week</i>
+
+`Content Curators` will one day be essential for ensuring that the petabytes of media items uploaded to Joystream are formatted correctly and comprehensively monitored and moderated. Our upcoming testnet allows this content monitoring to take place by giving users who are selected for the role administrative access to the Joystream content directory to make changes where necessary.
+
+We will have five slots available for `Content Curators` when Rome is launched. They will need to apply to the `Content Curator Lead` (a `sudo` role operated by [Jsgenesis](https://jsgenesis.com/)) and will have their application assessed by this lead before being added as a curator for a default term of one week.
+
+If successful in their application, curators will be rewarded with a flat reward of `$60` per week, resulting in a maximum pool of `$300` if all five content curators are signed up. If there are less than 100 pieces of content uploaded in the week in which a curator is operating, the individual payout will be reduced to `$30` for that week, resulting in a total pool of `$150`.
+
+There will also be two bonuses of `$50` and `$25` per week for the two most active curators respectively in terms of monitoring the items in the content directory for compliance with our metadata, category and content rules. These bonuses will be awarded by the `Content Curator Lead`. In total, during a typical week, `Content Curators` will be competing for their share of `$375` in XMR.
+
+Payouts will be made every Monday at ~11:00GMT.
+
+Storage Providers
+-----------------
+
+![](Storage%20Providers.png)
+
+<i>Compete for $300 per week, plus bonus for data</i>
+
+Unlike `Validators` that can come and go without too much friction (at least for now), a new `Storage Provider` replicates the entire content directory. As a consequence, the platform needs some stability for this role to avoid a poor user experience, or worse, loss of data. After migrating to the Rome testnet, the users that currently hold the role will get "kicked" out, and will have to sign up again if they still want to keep their role.
+
+Up to 10 `Storage Providers` that keep a full copy and provides continuous service for at least 24h will compete for `$300 per week`. In addition,  you will earn `$0.025/GB/week` (of data in the content directory) calculated on an average basis. You will also need a domain to point to your node, so unless you already own one (that you don't mind using), you will need to buy one.
```

Comment:
Let's start out with 8, but add that we will be far heavier on the hammer this time. Perhaps the curators could also be responsible for reporting on poor service?